perfkitbenchmarker/scripts/database_scripts/plot_sysbench_results.py

```diff
@@ -13,10 +13,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Plots per second output from STDERR files.
-How to use:
+"""Plots per second output from mysql_service_benchmark STDERR files.
+Prerequisites/assumptions:
+Gnuplot must be installed on the machine to use this module.
+  To check if gnuplot is installed on the machine enter:
+    'dpkg -l gnuplot'
+  To install gnuplot:
+    sudo apt-get install gnuplot-x11
+mysql_service_benchmark Stderr file
+  Can be generated with launch_mysql_service. If enabled, per_second_graphs flag
+  will automatically call this module and generate the per second graphs.
+no sysbench_warmup time
+  Parsing of file assumes sysbench_warmup_seconds is set to 0.
+Sysbench version
+  Assumes Sysbench GPL v2 (https://github.com/akopytov/sysbench) stderr output.
```

Comment:
sysbench GPL v2 is not the version number of sysbench

(GPL v2) is the version number of the licence.

The version of the sysbench should be 0.5 (as we just checked at your workstation).